internal/platform/config/config.go

```diff
@@ -70,6 +78,7 @@ func unmarshalNested(data []byte, cfg *Config) error {
 	err = json.Unmarshal(data, &cfg.Storage)
 	err = json.Unmarshal(data, &cfg.SpyNode)
 	err = json.Unmarshal(data, &cfg.NodeStorage)
+	err = json.Unmarshal(data, &cfg.RpcNode)
```

Comment:
This is a copy of the "flat json" idea which meant this had to be changed anytime the config changed.

Just use "normal" json. Take a look at nexus-api

Reply:
We have a separate ticket to update the configs to use a common system. https://tokenized.atlassian.net/browse/NEX-851